src/Servers/IIS/AspNetCoreModuleV2/Symbols/Microsoft.AspNetCore.ANCMSymbols.csproj

```diff
@@ -0,0 +1,31 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
+    <IncludeBuildOutput>false</IncludeBuildOutput>
+  </PropertyGroup>
+
+  <Import Project="..\..\build\assets.props" />
+
+  <ItemGroup Condition="'$(BuildIisNativeProjects)' == 'true' AND '$(VCTargetsPath)' != ''">
+    <NativeProjectReference Include="$(MSBuildThisFileDirectory)..\..\AspNetCoreModuleV2\OutOfProcessRequestHandler\OutOfProcessRequestHandler.vcxproj" Platform="x64" />
+    <NativeProjectReference Include="$(MSBuildThisFileDirectory)..\..\AspNetCoreModuleV2\OutOfProcessRequestHandler\OutOfProcessRequestHandler.vcxproj" Platform="Win32" />
+    <NativeProjectReference Include="$(MSBuildThisFileDirectory)..\..\AspNetCoreModuleV2\AspNetCore\AspNetCore.vcxproj" Platform="x64" />
+    <NativeProjectReference Include="$(MSBuildThisFileDirectory)..\..\AspNetCoreModuleV2\AspNetCore\AspNetCore.vcxproj" Platform="Win32" />
+  </ItemGroup>
+
+  <Target Name="AddPackNativeComponents" BeforeTargets="_GetPackageFiles" Condition="$(PackNativeAssets) == 'true'">
+    <ItemGroup>
+      <None Include="%(ShimComponents.DllLocation)"
+            Pack="true"
+            PackagePath="runtimes/win-%(ShimComponents.Platform)/native/$(NativeAssetsTargetFramework)" />
+      <None Include="%(ShimComponents.PdbLocation)"
+            Pack="true"
+            PackagePath="runtimes/win-%(ShimComponents.Platform)/native/$(NativeAssetsTargetFramework)" />
+    </ItemGroup>
+  </Target>
+
+  <!-- This project does not produce a binary. Disable the compiler. -->
+  <Target Name="Compile" />
```

Comment:
Can you verify there isn't actually a .dll getting produced? I suspect that due to implicit top-bottom imports from Microsoft.NET.Sdk, you are actually still compiling a .dll.

If that's the case, the solution is to go from implicit to explicit imports from Microsoft.NET.Sdk so that you can override the compiler targets.

https://github.com/aspnet/AspNetCore/blob/117dd8cff4231c1567938d9835a22ff9c525a9ae/src/Analyzers/shared/FeatureDetection/Microsoft.AspNetCore.Analyzers.FeatureDetection.Sources.csproj#L1-L3
https://github.com/aspnet/AspNetCore/blob/117dd8cff4231c1567938d9835a22ff9c525a9ae/src/Analyzers/shared/FeatureDetection/Microsoft.AspNetCore.Analyzers.FeatureDetection.Sources.csproj#L63-L67

Reply:
No dll: 
![image](https://user-images.githubusercontent.com/8302101/57885758-82105980-77e0-11e9-850f-1851eb769818.png)

 I do need to readd _._, otherwise I get the error `NuGet.Packaging.Core.PackagingException: Cannot create a package that has no dependencies nor content.`

Reply:
Weird that you'd get that issue. I'd have thought the `pdb` was sufficient content for that check.

Reply:
Do you see the dll in artifacts/bin/Microsoft.AspNetCore.ANCMSymbols/ ?